src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -699,11 +703,6 @@ export default class Fms {
      * @param runwayModel {RunwayModel}
      */
     replaceDepartureProcedure(routeString, runwayModel) {
-        // this is the same procedure that is already set, no need to continue
-        if (this.hasLegWithRouteString(routeString)) {
```

Comment:
I smell what you're stepping in.  But, I need to ask; what if you have the same `routeString` _and_ the same `runwayModel`?  Meaning, the user has issued the command for their current route (whether on purpose or accident).  

Then what?